examples/v2/matterbridge/config_chat2bridge.nim

```diff
@@ -66,14 +66,14 @@ type
       name: "metrics-server-port" .}: uint16
 
     ### Waku v2 options
-    staticnodesv2* {.
+    staticnodes* {.
       desc: "Multiaddr of peer to directly connect with. Argument may be repeated"
-      name: "staticnodev2" }: seq[string]
+      name: "staticnode" }: seq[string]
 
-    nodekeyv2* {.
+    nodekey* {.
       desc: "P2P node private key as hex"
       defaultValue: crypto.PrivateKey.random(Secp256k1, newRng()[]).tryGet()
```

Comment:
In case that no key is provided, do we display this default generated key to the user to use it for the next time?

Reply:
No, I think there could be security issues with displaying the private key, but we do display the listening address and peer ID generated from this key. For repeatability, though, users should generate, securely manage and provide their own keys.